src/main/java/uk/gov/justice/hmpps/casenotes/controllers/ControllerAdvice.java

```diff
@@ -22,16 +22,16 @@
 @Slf4j
 public class ControllerAdvice {
 
-    @ExceptionHandler(RestClientResponseException.class)
-    public ResponseEntity<byte[]> handleRestClientResponseException(final RestClientResponseException e) {
+    @ExceptionHandler(WebClientResponseException.class)
```

Comment:
cope with new webclient exceptions instead - not think not found should be `log.error`, but replicated existing functionality